_posts/2020-07-11-diversity-for-the-right-solution.md

```diff
@@ -0,0 +1,32 @@
+---
+layout:      posts
+background:  shortBackground
```

Comment:
Looks like the subtext gets cut off with shortBackground and mid screen resolutions. Suggest changing this to `mediumBackground`.
![image](https://user-images.githubusercontent.com/8344081/87225682-7a046900-c354-11ea-8da1-c3b1fbe581aa.png)

Reply:
I started with mediumBackground but the `posted by` line gets hidden then...

Reply:
Oh ugh, I just tried it and I see what you mean. Ok I revoke my suggestion! I can open a PR for fixing that spacing in general.

Reply:
i can add a new style that is in between the two in order to fix it for now and we can use your issue for a more permanent fix :)